Cargo.toml

```diff
@@ -13,6 +13,13 @@ Semantic version parsing and comparison.
 
 [dependencies]
 semver-parser = "0.6.1"
+regex = "0.1"
```

Comment:
I'm thinking maybe split the binary into separate project, just for the binary. This way you don't pull the dependencies with the library. (Although you'll get them with semver-parser anyways :))